lib/context-util.c

```diff
@@ -139,18 +139,18 @@ tcti_tabrmd_init (void)
 
     rc = tss2_tcti_tabrmd_init(NULL, &size);
     if (rc != TSS2_RC_SUCCESS) {
-        LOG_ERR ("Failed to get size for TABRMD TCTI context: 0x%" PRIx32, rc);
+        LOG_ERR ("Failed to get size for TABRMD TCTI context: 0x%" PRIx32 "\n", rc);
```

Comment:
Sounds good to me. I'll do that.

BTW, do you want the patch that splits on stderr and stdout or do you prefer to keep LOG_INFO() printing to stderr? (I prefer stdout but have no strong opinion)